### PR TITLE
FAT regression test fix

### PR DESCRIPTION
**Pull request by himaninare**

### CODE REVIEW

File: src/test/resources/FeatureFiles/CCSLatest/ccs_fm2_technical_security.feature [hunk 1]
```diff
@@ -269,24 +281,27 @@ Feature: GM Journey for FM2 framework (Technical and Security journey branch) ->
     When User clicks on the "Start now" button
     And  User selects what type of security "<secTypeRadio>" is needed radio button
     And  User clicks on the "Continue" button
+    And  User selects what type of security "<secTypeRadio1>" is needed radio button
+    And  User clicks on the "Continue" button
```
Comment (amrutha-sireesh) [inline src/test/resources/FeatureFiles/CCSLatest/ccs_fm2_technical_security.feature:285]: Please use
User selects which "<areaSuits>" your requirements instead

**Review by amrutha-sireesh — CHANGES_REQUESTED** [review]

File: src/test/resources/FeatureFiles/CCSLatest/ccs_fm2_technical_security.feature (hunk 1 above)
Comment (himaninare) [reply src/test/resources/FeatureFiles/CCSLatest/ccs_fm2_technical_security.feature]: Changes made

File: src/main/java/com/scale/businessPages/E2EPage.java [hunk 2]
```diff
@@ -814,12 +813,16 @@ public void checkErrorHandlingContent(String expectedErrorHandlingContent, Strin
...
         // TODO: code cleanup - common javascript methods needs to be implemented
         JavascriptExecutor executor = (JavascriptExecutor)driver;
-        executor.executeScript("arguments[0].scrollIntoView();", backCTA);
-        executor.executeScript("arguments[0].click();", backCTA);
+        if(!backCTA.isDisplayed()) {
+            executor.executeScript("arguments[0].scrollIntoView();", backCTA);
+        }else {
+            executor.executeScript("arguments[0].click();", backCTA);
+        }
+        waitForSeconds(5);
```
[the remark this answers is not in the document]
Comment (himaninare) [reply src/main/java/com/scale/businessPages/E2EPage.java]: Changes made

File: src/test/resources/FeatureFiles/CCSLatest/All_questioncontent_errorhandling.feature [hunk 3]
```diff
@@ -80,8 +89,8 @@ Feature: Scenarios for checking all questions and errors handling content for ea
     Then User checks Routes to Market Definitions content on "routesToMarketContent"
 
     Examples:
-      | ScenarioID | framework | sectorRadio1       | sectorRadio2 | serviceArea1 | serviceArea2  | location         |
-      | Scenario 1 | legal     | Central Government | Devolved     | Property     | Anything else | England or Wales |
+      | ScenarioID | framework |area| sectorRadio1       | sectorRadio2 | serviceArea1 | serviceArea2  | location         |
+      | Scenario 1 | legal     | Legal |Central Government | Devolved     | Property     | Anything else | England or Wales |
```
Comment (amrutha-sireesh) [inline src/test/resources/FeatureFiles/CCSLatest/All_questioncontent_errorhandling.feature:94]: Himani, Can you please apply 'area' changes to all relevant places?

File: config/ApplicationConfig.properties [hunk 4]
```diff
@@ -1,5 +1,5 @@
 browserName=CHROME
-baseURL=https://webuat.crowncommercial.gov.uk/
+baseURL=https://webdev.crowncommercial.gov.uk/
```
Comment (amrutha-sireesh) [inline config/ApplicationConfig.properties:2]: Do you want this change?

**Review by amrutha-sireesh — APPROVED** [review]

**Pull request merged**